index.html

```diff
@@ -116,14 +128,14 @@ <h2 id="p-title">Tonic</h2>
                 </div>
             </div>
         </div>
-        <div class="cards">
+        <div class="cards rev-2">
             <div class="project-image">
-                <img src="images/cardImg2.png" class="portfolio-pic"   alt="portfolio picture">
+                <img src="images/cardImg2.png" class="portfolio-pic2"   alt="portfolio picture">
             </div>
             <div class="project-content">
                 <div class="project-info">
                     <div class="project-name">
-                        <h2 id="p-title">Multi-Post<br>Stories</h2>
+                        <h2 id="p-title">Multi-Post Stories</h2>
                     </div>
                     <ul class="p-info">
                         <li class="company">CANOPY</li>
```

Comment:
Please wrap your images inside the li tag in a tag. This will make your links more dynamic and link to the intended actions